components/ToDos/ToDos.js

```diff
@@ -0,0 +1,27 @@
+import getAllTasks from "@/lib/data";
```

Comment:
```suggestion
import {initialTodos} from "@/lib/data";
```

Please, have a look at my comment in your `data.js` file.

Reply:
Additionally, I recommend to keep the naming consistent, e.g. `task` and `taskList` or `todo` and `todoList`. I would not mix both to avoid confusion. Therefore I suggest to rename this component to `TaskList`.